src/app/header/header.component.ts

```diff
@@ -32,8 +32,21 @@ export class HeaderComponent implements OnInit {
 
   menuCallbacks = new Map<String, MenuHiddenCallback>([
     [
-      'settings', function (headerComponent, context) {
-        return headerComponent.loggedInUser !== context.user;
+      '_settings', function (headerComponent) {
+        return Observable.combineLatest(
+          headerComponent.contexts.current.map(val => val.user.id),
+          headerComponent.userService.loggedInUser.map(val => val.id),
+          (a, b) => (a !== b)
+        );
+      }
+    ],
+    [
+      '_resources', function (headerComponent) {
+        return Observable.combineLatest(
```

Comment:
This function and the one above it appear to do the same thing. Do you plan to make them different.